scripts/installer/install.sh

```diff
@@ -392,6 +392,7 @@ mount -t $fs_type -o defaults,rw $bisdn_linux_dev $bisdn_linux_mnt || {
 }
 
 # install fs
+printf "%s" "Writing root file system: " >&2
```

Comment:
This isn't an error, so it should go to stdout, not stderr.

And you can use `echo -n` to prevent echo from adding a newline at the end.

Reply:
Fixed.

Reply:
Thinking about it a bit more, I don't think we should try to have both on one line, and having it in two lines is fine.

Also we don't need to emulate mkfs.ext4 (or whatever is printing the previous line), and do our own thing. e.g.

```
Writing root file system to /dev/sda2 ...
Finished writing root filesystem.
```

Reply:
Done.